### ISSUE TITLE
HTTP 411 when upload behind a squid proxy

### ISSUE THREAD

**Issue by rafaelsisweb** [minio/minio-java#489]
We are using minio-java to upload some files to S3. Today we got some cases when our client runs behind a "squid proxy" that work when upload small files, but on large files upload got fails with an XmlPullParserException, and tracing the requests we got and request error HTTP 411.
It seems related to the fact tha large files will use multipart upload.
Is there any way to force large files to use single upload?

```
s3MinioClient.putObject(S3Upload.getBucketName(), keyName, file.getCanonicalPath());
```

**Comment by vadmeste** [comment]
HTTP 411 is content length, looks like minio-java is not sending content-length with multipart uploads.

I am taking this.

**Comment by GarimaKapoor** [comment]
@vadmeste any updates on this issue?

**Comment by balamurugana** [comment]
@rafaelsisweb Its fixed by minio/minio-java#490.  Could you verify with release 3.0.3?

**Comment by harshavardhana** [comment]
Closing this as fixed in latest release 3.0.3
